### ISSUE TITLE
KeyError: 'cstring' when calling setDropdown

### ISSUE THREAD

**Issue by zachlipton** [bertrandmartel/tableau-scraping#2]
Hi! Not sure how much you're maintaining this, but I was playing around with the library for some tableau scraping of COVID data and came across an issue.

Everything worked fantastic until I tried to use `setDropdown()` on [this dashboard](https://tableau.azdhs.gov/views/ELRv2testlevelandpeopletested/TestsConducted). When I do that, I get:

```/usr/local/lib/python3.6/dist-packages/tableauscraper/utils.py in getWorksheetCmdResponse(selectedZone, dataFull)
    201 
    202 def getWorksheetCmdResponse(selectedZone, dataFull):
--> 203     cstring = dataFull["cstring"]
    204     details = selectedZone["presModelHolder"]["visual"]["vizData"]
    205 

KeyError: 'cstring'
```

I think this is happening because the data in question is integer data instead of strings, but I'm not entirely sure what's going on. 

I have a testcase that demonstrates the issue at https://colab.research.google.com/drive/14nUWYCvJB1ERsROIwY-AL_ZkIE73GrMU

**Comment by bertrandmartel** [comment]
@zachlipton Hello, sorry for the late response, I've fixed the dropdown script example which should work on sample url provided. Unfortunately, you're link is not accessible anymore, can you upgrade to the latest version 0.0.7 and check if you still experience this issue ?

**Comment by bertrandmartel** [comment]
@zachlipton a similar bug #5 was also resolved in v0.0.8
